workspace-hack/Cargo.toml

```diff
@@ -14,45 +14,41 @@ publish = false
 ### BEGIN HAKARI SECTION
 [dependencies]
 axum = { version = "0.6" }
-bitflags = { version = "2", default-features = false, features = ["std"] }
 bytes = { version = "1" }
 clap = { version = "4", features = ["derive"] }
 crypto-common = { version = "0.1", default-features = false, features = ["std"] }
 digest = { version = "0.10", features = ["mac", "std"] }
-either = { version = "1" }
+either = { version = "1", default-features = false, features = ["use_std"] }
 futures-channel = { version = "0.3", features = ["sink"] }
 futures-util = { version = "0.3", features = ["channel", "io", "sink"] }
 getrandom = { version = "0.2", default-features = false, features = ["js", "rdrand", "std"] }
-itertools = { version = "0.11" }
 libc = { version = "0.2", features = ["extra_traits"] }
 log = { version = "0.4", default-features = false, features = ["std"] }
-madsim-tokio = { version = "0.2", default-features = false, features = ["fs", "io-util", "macros", "net", "rt", "rt-multi-thread", "signal", "sync", "time"] }
-madsim-tonic = { version = "0.4", default-features = false, features = ["tls"] }
+madsim-tokio = { git = "https://github.com/Phoenix500526/madsim.git", branch = "update-tonic", default-features = false, features = ["fs", "io-util", "macros", "net", "rt", "rt-multi-thread", "signal", "sync", "time"] }
+madsim-tonic = { git = "https://github.com/Phoenix500526/madsim.git", branch = "update-tonic", default-features = false, features = ["tls"] }
```

Comment:
Please add comments here explaining what are the modifications of this branch.

Reply:
The `workspace-hack/Cargo.toml` is generated.